taskipy/pyproject.py

```diff
@@ -10,29 +10,47 @@
     MissingPyProjectFileError,
     MissingTaskipyTasksSectionError
 )
+from taskipy.variable import Variable
```

Comment:
May I ask what makes it more organized for you? Keeping the imports sorted in alphabetical order generally seems to be the preferred style to be able to quickly find imports. If we're taking that as the preferred style, `taskipy.variable` would be at the end. So that's where that rationale comes from.

Reply:
Just because the errors import is really big 😅

Reply:
I don't agree with that line of reasoning, but I can give you this one :smile: